plugin/src/main/java/org/jqassistant/tooling/dashboard/plugin/impl/mapper/ContributionMapper.java

```diff
@@ -0,0 +1,14 @@
+package org.jqassistant.tooling.dashboard.plugin.impl.mapper;
+
+import org.jqassistant.tooling.dashboard.api.dto.ContributionDTO;
+import org.mapstruct.Mapper;
+import org.mapstruct.factory.Mappers;
+
+@Mapper(uses = FileMapper.class)
+public interface ContributionMapper {
```

Comment:
Please verify if this mapper is actually used. If this is not the case then this class should be deleted.